src/libraries/System.Data.Common/src/System/Data/Common/DataAdapter.cs

```diff
@@ -502,6 +503,14 @@ protected virtual int Fill(DataTable[] dataTables, IDataReader dataReader, int s
             }
         }
 
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:RequiresUnreferencedCode",
+            Justification = "chapterValue is not used here")]
```

Comment:
Let's use the full name of the parameter, so it is obvious what we are referring to.

```suggestion
            Justification = "parentChapterValue is not used here")]
```

This is repeated elsewhere.